src/test/java/com/coravy/hudson/plugins/github/GithubProjectPropertyTest.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2015 CloudBees, Inc.
```

Comment:
Uhh, sorry. Seems I didn't follow-up to https://github.com/jenkinsci/github-plugin/commit/97f1c9dd76f6af91da4dfb6f343fda6de8a2727f